src/lib.rs

```diff
@@ -1608,6 +1608,38 @@ pub trait Itertools : Iterator {
         None
     }
 
+    /// Returns `true` if the given item is present in this iterator.
+    ///
+    /// This method is short-circuiting. If the given item is present in this
+    /// iterator, the this method will consume the iterator up-to-and-including
```

Comment:
```suggestion
    /// iterator, this method will consume the iterator up-to-and-including
```

Typo. Let's see if the mobile app will allow me to correct it in this manner...